headers/functions/arm9.h

```diff
@@ -1082,6 +1083,8 @@ void DeleteAllPossibleDungeonsList(void);
 int GenerateAllPossibleDeliverList(void);
 void DeleteAllPossibleDeliverList(void);
 void ClearMissionData(struct mission* mission);
+bool ValidateNormalChallengeMission(struct mission_rescue_bin* param_1, struct mission* mission);
```

Comment:
It seems like we could have a better name than `param_1`, given that we know what it is?